easybuild/easyconfigs/t/Tcl/Tcl-8.6.3-intel-2015a.eb

```diff
@@ -12,6 +12,10 @@ toolchain = {'name': 'intel', 'version': '2015a'}
 source_urls = ["http://prdownloads.sourceforge.net/tcl"]
 sources = ['%(namelower)s%(version)s-src.tar.gz']
 
+dependencies = [
+    ('zlib', '1.2.8'),
```

Comment:
Hmm, we would need to add this in the other Tcl easyconfigs too, but this can be a can of worms, since the zlib version used here has to match the zlib used by other entries in the dependency tree where `Tcl` is a part of...

@gjbex: Thoughts? Should we (try to) add this to the other Tcl easyconfigs too (separate PR, for sure)?